groups/nts/ntsu/ntsu_socketutil.cpp

```diff
@@ -1198,17 +1361,11 @@ ntsa::Error SocketUtil::send(ntsa::SendContext*       context,
     iovec.iov_base = const_cast<void*>(data);
     iovec.iov_len  = size;
 
-    msghdr msg;
-    bsl::memset(&msg, 0, sizeof msg);
-
-    if (specifyEndpoint) {
-        msg.msg_name    = &socketAddress;
-        msg.msg_namelen = socketAddressSize;
-    }
-
     msg.msg_iov    = &iovec;
     msg.msg_iovlen = 1;
 
+    context->setBytesSendable(size);
+
     ssize_t sendmsgResult =
```

Comment:
sendmsgResult as a counter of bytes does not include control message bytes, right?

Reply:
No, just real data.